packages/carbon-web-components/src/components/pagination/pagination.ts

```diff
@@ -360,17 +371,22 @@ class CDSPagination extends FocusMixin(HostListenerMixin(LitElement)) {
           <slot @slotchange=${handleSlotChange}></slot>
         </cds-select>
         <span
+          part="pagination-items-count"
           class="${prefix}--pagination__text ${prefix}--pagination__items-count"
           >${this._renderStatusText()}</span
         >
       </div>
-      <div class="${prefix}--pagination__right">
-        <label for="select" class="${prefix}--label ${prefix}--visually-hidden">
+      <div part="right-container" class="${prefix}--pagination__right">
+        <label
+          part="pagination-right-label"
+          for="select"
+          class="${prefix}--label ${prefix}--visually-hidden">
           ${formatLabelText({ count: totalPages })}
         </label>
         ${pagesUnknown || !totalItems
           ? html`
               <span
+                part="pagination-text-right"
```

Comment:
Just below here from lines 394-424, again, missing `part` attributes on `<cds-select>` and `<cds-select-item>` elements, both part of the shadow root for this component.